data/strings/sound.txt

```diff
@@ -429,6 +429,92 @@
     zh-Hans = 出口.
     zh-Hant = 出口.
 
+  [route_beginning]
+    en = Follow the highlighted route.
+    af = Volg die gemerkte roete.
+    ar = اتبع الطريق المميز.
+    be = Ідзіце па вылучаным маршруце.
+    bg = Следвайте маркирания маршрут.
+    ca = Seguiu la ruta destacada.
+    cs = Postupujte podle zvýrazněné trasy.
+    da = Følg den fremhævede rute.
+    de = Folgen sie der markierten route.
+    el = Ακολουθήστε την επισημασμένη διαδρομή.
+    es = Siga la ruta resaltada.
+    et = Järgige esiletõstetud marsruuti.
+    eu = Jarraitu nabarmendutako ibilbidea.
+    fa = مسیر مشخص شده را دنبال کنید.
+    fi = Seuraa korostettua reittiä.
+    fr = Suivez l'itinéraire mis en évidence.
+    he = עקבו אחר המסלול המודגש.
+    hi = हाइलाइट किए गए मार्ग का अनुसरण करें.
+    hr = Pratite označenu rutu.
+    hu = Kövesse a kiemelt útvonalat.
+    id = Ikuti rute yang disorot.
+    it = Seguire il percorso evidenziato.
+    ja = 強調表示されたルートに従ってください。
+    ko = "강조 표시된 경로를 따르세요."
+    lt = Sekite pažymėtą maršrutą.
+    mr = हायलाइट केलेल्या मार्गाचे अनुसरण करा.
+    nb = Følg den uthevede ruten.
+    nl = Volg de gemarkeerde route.
+    pl = Podążaj zaznaczoną trasą.
+    pt = Siga a rota destacada.
+    ro = Urmați traseul evidențiat.
+    ru = Следуйте по выделенному маршруту.
+    sk = Postupujte podľa zvýraznenej trasy.
+    sv = Följ den markerade rutten.
+    sw = Fuata njia iliyoangaziwa.
+    th = ไปตามเส้นทางที่ไฮไลท์
+    tr = Vurgulanan rotayı takip edin.
+    uk = Дотримуйтеся виділеного маршруту.
+    vi = Đi theo tuyến đường được đánh dấu.
+    zh-Hans = 遵循突出显示的路线。
+    zh-Hant = 遵循突出顯示的路線。
+
+  [route_recalculating]
+    en = Rerouting.
```

Comment:
rerouting might be too short, people might not hear it. It's also quite technical language which we should avoid imo

my car's satnav says "Recalculating route" which IMO would be a better option :)

Reply:
This is exactly the what I suggested to be used in Portuguese (literal translation of "Recalculating route"), because this is what I also hear on my other apps.

Reply:
> This is exactly the what I suggested to be used in Portuguese (literal translation of "Recalculating route"), because this is what I also hear on my other apps.

oh yeah! oops 😆

Reply:
I used "recalculating" at first which I know is standard, but I considered that it's a bit unfriendly (are we using a calculator, or a navigator? Is there math involved? Are we bad drivers for making our GPS do so much math? The more it talks at us the worse we feel about our bad driving and this bad traffic.)

I wonder if a small chime is actually better than words for this, I can remember many times being frustrated at GPS for "recalculating" too much (like let's say if you're taking a detour to a gas station that's a bit off the highway but are too flustered to press buttons on the GPS) and the noise is more irritating than helpful. All we're looking for is a small reminder that the guidance has changed, we're not really communicating any substantial information. The only danger in the reminder being unheard is if the next turn is many miles away and so the GPS silently lets you continue on for a long time, but that can be solved in other ways (like in #2802 where intermediary "...continue for 232 miles" may be a good idea, which doesn't need to necessarily be announced immediately upon recalculation)

Reply:
Recalculating the route is a better option considering that it takes some time. Google doesn't say it because it does it (almost) instantly on fast servers.